extensions/vertx-http/deployment/src/main/java/io/quarkus/vertx/http/deployment/FrameworkRootPathBuildItem.java

```diff
@@ -0,0 +1,33 @@
+package io.quarkus.vertx.http.deployment;
+
+import io.quarkus.builder.item.SimpleBuildItem;
+
+public final class FrameworkRootPathBuildItem extends SimpleBuildItem {
+    private final String frameworkRootPath;
+    private final boolean separateRoot;
+
+    public FrameworkRootPathBuildItem(String frameworkRootPath) {
+        this.frameworkRootPath = frameworkRootPath;
```

Comment:
That only configure the "/q".
Don't you want to allow having a separate HTTP server (maybe it's done somewhere else)

Reply:
I've created https://github.com/quarkusio/quarkus/issues/13602 to handle the port and binding options to keep this PR simple and get it in

Reply:
ah ok!